tests/python/pants_test/engine/test_rules.py

```diff
@@ -780,19 +772,18 @@ def a() -> A:
     subgraph = self.create_subgraph(A, rules, SubA(), validate=False)
 
     self.assert_equal_with_printing(
-      dedent(
-        f"""\
-        digraph {{
-          // root subject types: SubA
-          // root entries
-            "Select(A) for ()" [color=blue]
-            "Select(A) for ()" -> {{"{fmt_rule(a)} for ()"}}
-          // internal entries
-            "{fmt_rule(a)} for ()" -> {{}}
-        }}"""
-      ).strip(),
+      dedent(f"""\
```

Comment:
Nit: I originally had the `f"""\` part on a new line because this is how Black would format things (I know we haven't officially adopted Black, but it seems like we're on track to adopt it in some variety so I've been using the standard).

I won't block on this by any means and I know it's tedious to change back, only wanted to explain the original rationale.

Reply:
Oh, I'll fix that! It wasn't intentional!